STYLEGUIDE.adoc

```diff
@@ -30,10 +25,35 @@ ____
 As opposed to "the Jenkins cluster", "agent pool" or any other phrase to
 describe the collective set of resources provided by Jenkins.
 
+
+=== Pipeline
+
+When referring to a Jenkins Pipeline via short-hand ("Pipeline"), it
+should always be title-cased. When referring to a conceptual pipeline
+("continuous delivery pipeline"), it it should always be lower-cased.
+
+With the
+link:https://jenkins.io/blog/2016/12/19/declarative-pipeline-beta/[Declarative Pipeline beta]
+there are two differen syntaxes for Pipeline which must be referred to when
+writing about the subject. When referring to the entire system, "Jenkins
+Pipeline" or "Pipeline" should be used. When referring to a specific sytnax,
+the following should be used
+
+
+Scripted Pipeline::: functionality defined by the fully-featured, expressive,
```

Comment:
"the fully-featured, expressive, Groovy-based DSL"

Both are fully-featured Groovy-based DSLs.  They are both subsets of Groovy. 

How about: 
"the lower-level Groovy-based DSL that accepts a larger subset of the Groovy language.  This allows it to behave more like a general purpose scripting language."